### PR TITLE
images/kube-proxy: update to 4.2, add iptables wrappers

### PR DESCRIPTION
**Pull request by squeed**

### CODE REVIEW

**Comment by squeed** [comment]
/test images

Want to verify that everything builds correctly

**Comment by squeed** [comment]
/test images

Wow, namespaces are cleaned up quickly.

**Comment by squeed** [comment]
What the heck - the hyperkube base image seems to not have hyperkube?

File: images/kube-proxy/Dockerfile.rhel [hunk 1]
```diff
@@ -1,9 +1,17 @@
-FROM registry.svc.ci.openshift.org/ocp/4.0:hyperkube
+FROM registry.svc.ci.openshift.org/ocp/4.2:hyperkube
 
-RUN INSTALL_PKGS="iptables conntrack-tools" && \
+RUN INSTALL_PKGS="conntrack-tools" && \
     yum install -y --setopt=tsflags=nodocs $INSTALL_PKGS && \
     yum clean all && rm -rf /var/cache/*
 
+COPY ./images/sdn/scripts/iptables /usr/sbin/
```
Comment (DanyC97) [inline images/kube-proxy/Dockerfile.rhel]: @squeed for my own knowledge (if you don't mind), what is the rational of having this iptables scripts?

Comment (squeed) [reply images/kube-proxy/Dockerfile.rhel]: We don't know what mode the host's iptables is in - is it nftables compat mode, or legacy iptables mode? So, we just escape via chroot to the host and execute iptables there.

**Comment by squeed** [comment]
hey @JacobTanenbaum can you take a look?

**Pull request merged**